libs/gi/db/src/Database/migrate.ts

```diff
@@ -172,6 +172,8 @@ export function migrateGOOD(good: IGOOD & IGO): IGOOD & IGO {
       const optConfig =
         buildSettings.find(({ id }: { id: string }) => id === characterKey) ??
         {}
+      // Only migrate characters with either a mtarget or an opttarget
+      if (!customMultiTarget?.length && !optConfig.optimizationTarget) return
```

Comment:
Not sure if we even need the opt target thing. Won't this still end up creating 15 loadouts if a support is in 15 teams and has a opt target?
Or am I reading the logic wrong and this will just create one loadout?

Reply:
merging loadouts is complicated logic that probably can't be safely implemented.

This logic avoids creating teams/loadouts for characters that are in GO, but don't have a opt-target selected. 
If someone has like 60 characters, but only set up optimizer for like 20 characters, it will leave the other 40 characters that are not in any teams of the core 20 characters to be "no loadout/team"